.gitignore

```diff
@@ -1,17 +1,9 @@
 node_modules
 coverage
 dist
-.opt-in
-.opt-out
 .DS_Store
-.eslintcache
-.idea/
 
 # these cause more harm than good
 # when working with contributors
 package-lock.json
 yarn.lock
-
-# cypress recordings during a test run are temporary files
-/cypress/videos/
-/cypress/screenshots/
```

Comment:
These should be ignored. If someone runs `npm test`, a video will be created. If a test fails, a screenshot will be created. These file will either have to be manually deleted or accidentally committed.

Reply:
haha, whoops. @MichaelDeBoey has been helping me clean up some repos and I think this was probably a copy/paste error and I just didn't review well enough. I'll fix this.

Reply:
Sorry, my bad 😕